### PR TITLE
Cleaned up task2_getting_started

### PR DESCRIPTION
**Pull request by zhixianliding**

### CODE REVIEW

**Comment by im-a-jin** [comment]
Do you mind making the following changes?

Top level:
- Change "%cd MTNeuro/MTNeuro" to just "%cd MTNeuro" and the relevant imports to "MTNeuro.bossdbdataset" etc.
- Clear the outputs on the pip install cell
- Get rid of the random seeding
- Do `from tqdm.notebook import tqdm` instead of `from tqdm import tqdm`

Dataset Description
- Do you mind clarifying the train/test split a bit more (see https://github.com/MTNeuro/MTNeuro/blob/347a192cc872023f4a685561c73f64b78fadb306/notebooks/task1_getting_started.ipynb)
- Can you update each [1] so that it links to the associated reference? (an example of how to do it should be in the above notebook if you download it)
- Since we're cding a directory up set root to "./MTNeuro/"

Data Visualization
- Remove the type and min/max values of the input/annotation. I don't think they're really necessary. You can keep the shape and possible classes (on a new line though)

Building the Model:
- Cite UNet as [3] since you use [1] and [2] in the earlier sections
- Align the indents of the UNet and dataloader params (really nitpicky, sorry)

Training the Model:
- Remove the `lr_scheduler` param since it defaults to None anyway
- Align trainer indentation

Evaluating the Model:
- Add a docstring for the predict function
- Use torch.inference_mode instead of no_grad in predict
- Align inline comment indentation
- Either send the output to the device or explicitly to the cpu and remove the "can also do this on cpu" comment
- Don't need no_grad in the prediction loop since the predict function already has it
- In the "calculate..." comment say that we'll be using segmentation_models_pytorch to calculate the stats
- Get rid of batch_iter and directly index test_dataloader (reference the revised task 1 notebook)

Visualizing Model Predictions:
- Move the comments about using the same cmap and legends into the markdown cell right above the visualization code.
- Put average F1 and IOU on new lines after the new scores
- The accuracy equation should be tp+tn not tp+fp
- Maybe define specificity, sensitivity, precision, and recall? It might get a little clunky and I assume most people using the notebook would know those terms so this is up to you.


Use two hashtags (##) for each major header and four (####) for each minor header. Major headers would be stuff like "Building the Model", "Training the Model", etc. You can reference the revised task 1 notebook for more examples.

**Comment by erikjohnson24** [comment]
Mostly looks good- see a couple remaining changes still required from the list above- Accuracy eq should be tp + tn, not tp + fp, lr schedule can still be removed.

**Review by erikjohnson24 — CHANGES_REQUESTED** [review]
please address the last couple comments, then good to go I think

**Comment by zhixianliding** [comment]
I pushed the changes I made to update the accuracy formula and to remove lr schedule; it was my fault that I didn't see Matthew's updated comments at the time of editing the notebook. I realized that many of my images on the notebook are hosted locally on branch notebook_cleanup. I think a good time to update the links to the images is when we merge notebook_cleanup with main, so I can use the image links on main. If I update the image links now, I'll need to immediately push the images to main. Is that better than updating the images later?

**Comment by erikjohnson24** [comment]
Got it- I will approve then go ahead and merge, then push the updated links directly to main? I think fine for this case!

**Review by erikjohnson24 — APPROVED** [review]
